.npm/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "standard",
+    "rules": {
+        "no-console":0
+    }
+}
```

Comment:
minor nitpick, but we should add newline at end of file 😋

Reply:
Gonna fix that 👍